source/assets/scripts/backend.js

```diff
@@ -66,7 +66,8 @@ function recalculate_intolerances(recipe){
       }
     }
   }
-  // if (recipe.intolerances.includes('Soy-free')) tag['Soy-free']=true; 
+  // if (recipe.intolerances.includes('Soy-free')) tag['Soy-free']=true;
+  if (tag['Vegan']) tag['Vegetarian'] = true;
```

Comment:
saw a bunch of instances where a vegan thing isn't vegetarian so decided to fix it